packages/wxt/src/core/utils/eslint.ts

```diff
@@ -0,0 +1,16 @@
+import { resolve } from 'node:path';
+import fs from 'fs-extra';
+import { wxt } from '../wxt';
+
+export async function getEslintVersion(): Promise<string[]> {
+  try {
+    const packageJsonPath = resolve(
+      wxt.config.root,
+      'node_modules/eslint/package.json',
```

Comment:
Unfortunately, there is no guarentee that this file exists... Monorepos for example, may not have eslint installed in this extention's package. Instead, we should import it and grab the version from the imports. Something like this:

```ts
const { ESLint } = await import('eslint');
console.log(ESLint.version)
```

Reply:
ESLint is not a friendly import with the `"type": "module"` 

Looked for an alternative option, found the `createRequire` flow, found out that `createRequire` is already declared, so looked to see if and how that was handled and mirrored that to import ESLint, let me know if that works for you

Reply:
Hmm, I thought dynamic imports supported CJS modules. I see your `createRequire` code, if that's what's required, that's good with me 👍